<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>572</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTextEdit" name="inputText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="inputButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>250</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>sumbit</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="showText">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="showButton">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>480</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>show</string>
   </property>
  </widget>
  <widget class="QPushButton" name="fileBrowse">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
